### PR TITLE
[15.0][FIX] website_sale_product_attribute_filter_category: Removed products_attributes_collapsible template + migration script

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
The products_attributes_collapsible template has been removed because the functionality of the module is to add the attribute categories in the filter section. This template added the feature of collapsible attributes that should be independent.

A new module has been created to fold attributes independently: https://github.com/OCA/e-commerce/pull/740

cc @Tecnativa TT41345

@chienandalu @pedrobaeza could you check that everything is correct?

ping @stefan-tecnativa

### CODE REVIEW

File: website_sale_product_attribute_filter_category/migrations/15.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    env["ir.ui.view"].search(
+        [
+            (
+                "key",
+                "=",
+                "website_sale_product_attribute_filter_category"
+                ".products_attributes_collapsible",
+            )
+        ]
+    ).unlink()
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_category/migrations/15.0.1.0.0/pre-migration.py]: During upgrade we try to minimize ORM operations:
```suggestion
    openupgrade.logged_query(
        env.cr,
        """
        DELETE FROM ir_ui_view
        WHERE key = 'website_sale_product_attribute_filter_category.products_attributes_collapsible'
        AND website_id IS NOT NULL
        """,
    )
```

**Review by chienandalu — APPROVED** [review]
:+1:

File: website_sale_product_attribute_filter_category/migrations/15.0.1.0.0/pre-migration.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply website_sale_product_attribute_filter_category/migrations/15.0.1.0.0/pre-migration.py]: changes made! thanks for the suggestion :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Comment by chienandalu** [comment]
Oh it isn't fine! The migration script won't match the version!

**Comment by pedrobaeza** [comment]
There's no problem. The module was just merged, and this is for the migration path.

I launch again the command:

/ocabot merge major

**Pull request merged**